configure.ac

```diff
@@ -88,24 +88,41 @@ if test x"$enable_smtp" != xno ; then
 	)
 fi
 
-AC_MSG_CHECKING(whether to use libesmtp)
+AC_MSG_CHECKING(whether to use libesmtp >= v1.1.x)
```

Comment:
Can you please add a comment explaining the logic you described in this PR? That would certainly help future readers.

Reply:
You mean a comment in the configure.ac file itself?

Reply:
Yes, sorry for being unclear. It would help to understand why there is a complicated logic for libesmtp in configure.ac.

Reply:
That's fine. I added some comment lines making it more clear why there is a new block to check for libesmtp >= 1.1.0 and only use the older block to check for libesmtp < 1.1.0 if the newer version is not found.

Reply:
That's perfect, thanks for the change @tru7!
I'll merge it right away.